staging/prometheus-operator/templates/prometheus/rules/mesosphere-rules/velero-rules.yaml

```diff
@@ -0,0 +1,29 @@
+# Rules for alerting against velero
+{{- if and .Values.mesosphereResources.create .Values.mesosphereResources.rules.velero }}
```

Comment:
Let's be sure, we don't fire this alert if Velero is not running in the cluster. Let's remember that customer could disable the Velero Addon at any time

Reply:
This should work even if velero is enabled at any time.

@shaneutt ideally, dashboards and alerts should be enabled if the addon is enabled at any time. But that is a bit unrealistic :/

Reply:
yeah... think it's unrealistic for now.....

Reply:
we could have an operator that watches changes to the addon list, and so it disable/enable these settings on prometheus :)

Reply:
I've defaulted the rule to `false` here - I'm inclined to leave it defaulted to `false` on our end in kubeaddons-configs as well, and document somewhere that users can override it to `true` if they wish to turn on that rule for velero, with a caveat that they must turn it off if they ever disable velero.

Not ideal, but I think it's the best solution for now until we figure out how to dynamically configure dashboards/rules dependent on whether addons are enabled/disabled.